projects/plugins/jetpack/modules/tiled-gallery/tiled-gallery/tiled-gallery-rectangular.php

```diff
@@ -268,6 +289,54 @@ public function calculate_image_sizes( $group ) {
  * Jetpack tiled row class.
  */
 class Jetpack_Tiled_Gallery_Row {
+	/**
+	 * Groups of images
+	 *
+	 * @var array
+	 */
+	public $groups;
+
+	/**
+	 * Image ratio.
+	 *
+	 * @var int
```

Comment:
I have the feeling this (and weighted_ratio) should both be float. Same for the return values of the `get_ratio` and `get_weighted_ratio` functions.

Reply:
Done in b503fca7d2b44b8c41de15263072a4145cc1b69c